### ISSUE TITLE
WIFI-streaming mode: Get the video data with my own code(Python Webrtc)

### ISSUE THREAD

**Issue by Daretowin** [marek-simonik/record3d-wifi-streaming-and-rgbd-mp4-3d-video-demo#6]
I want to get the video data(mp4 or the byte stream) with my own code(Python Webrtc). I can connect to the app and get the camera pose but the video data is not avaliable. I need your help. PS: I have bought the app with 48￥.

**Comment by Daretowin** [comment]
non-existing PPS 0 referenced 错误是由于视频解码器不支持编码格式所引起的，可以尝试更换视频解码器，或者更换使用的编码格式。

**Comment by Daretowin** [comment]
I want to know the video formal, because the gpt-3 tell that the error is video code not support the code format.

**Comment by marek-simonik** [comment]
Based on info from `chrome://webrtc-internals`, Google Chrome uses H264 encoding for the Record3D data stream, so it is seems like it's possible to get Record3D to stream in H264 via WebRTC.

I think the codec used in WebRTC session can be negotiated in the SDP answer — how does your SPD answer look like? Perhaps your SPD answer contains a different codec than H264 (e.g. VP8)?

Camera pose streaming is not available for Wi-Fi streaming.

It sounds like you are not satisfied with Record3D, so I want to let you know that you can get your money back by asking Apple for refund: https://support.apple.com/en-us/HT204084
